php/class-plugin.php

```diff
@@ -163,6 +169,9 @@ function init() {
 		if ( $this->is_module_active( 'deferred_customize_widgets' ) ) {
 			$this->deferred_customize_widgets = new Deferred_Customize_Widgets( $this );
 		}
+		if ( $this->is_module_active( 'customize_settings_snapshot' ) ) {
+			$this->customize_settings_snapshot = new Customize_Settings_Snapshot( $this );
```

Comment:
OK, I'll make some changes to the structure before I move forward with the functionality.